examples/vpc-scenario-3/README.md

```diff
@@ -0,0 +1,50 @@
+# Example to test the VPC Scenario 3 Module
+
+## Environment creation and deployment
+
+This module creates a VPC with both public and private-vpn subnets spanning one or
+more availability zones, an internet gateway for the public subnets and a
+hardware VPN gateway for the private subnets.
+
+To use this example set up AWS credentials and then run the commands in the
+following order:
+
+Note that, when using this module and deploying the VPC for the first time,
+Terraform needs the user to add the VPC, Subnets, and then Route Tables. For
+example, use targets to apply these updates sequentially:
+
```

Comment:
Should I trust the contents of the README from here on out 100%? Is this how to deploy and test the env? I believe the README should be more explicit about the demo, what features will be demoed, how to setup the env, and how to carry out the demo. The features demoed should help the user focus on confirming the VPC network and VPN are setup correctly.